frontend/src/surveys/containers/GetSurveysList/GetSurveysList.js

```diff
@@ -0,0 +1,28 @@
+import { connect } from 'react-redux';
+
+import SurveysList from '../../components/SurveysList';
+import HttpUtility from '../../utils';
+// server.fetchDev(10);
+const getSurversFromServerSide = () => HttpUtility.GetSurveys();
+
+const getSurveys = state => state.surveys.concat(getSurversFromServerSide(state.server));
+
+const tempOnSurveyClick = () => {
+    alert('click');
+};
+
+const mapStateToProps = state => ({
+    surveys: getSurveys(state),
```

Comment:
mapStateToProps must not contain AJAX function calls. Review our previous comments